doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -60,6 +60,7 @@
  <!-- NOTE: Add new entries sorted by issue number to minimize the possibility of conflicts when merging. -->
 
  * Build failure for Windows 11 i686-pc-windows-msvc [#235](https://github.com/eclipse-iceoryx/iceoryx2/issues/235)
+ - 'win32call' needs to provide the last error [#241](https://github.com/eclipse-iceoryx/iceoryx2/issues/241)
```

Comment:
If you touch some code in this PR again, could you replace `-` with `*`. This is used all over the file and my inner daemon screaming for consistency is crying ... sorry ...